src/pages/thank-you.astro

```
@@ -0,0 +1,22 @@
+---
+import DefaultLayout from '@layouts/DefaultLayout.astro'
+import PageHeader from '@components/PageHeader.astro'
+import { Heading, Link, Notification } from 'accessible-astro-components'
```

ESLint

[error] 'Heading' is defined but never used.

(no-unused-vars)